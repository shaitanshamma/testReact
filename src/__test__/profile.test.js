import Profile from "../components/Profile";
import renderer from 'react-test-renderer';


describe('component test', () => {
    it('matches snapshot with profile', () => {
        const component = renderer.create(<Profile/>);
        expect(component).toMatchSnapshot();
    });
})