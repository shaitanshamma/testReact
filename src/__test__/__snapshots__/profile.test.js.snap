// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component test matches snapshot with profile 1`] = `
<div>
  Profile page
</div>
`;

exports[`reducer test matches snapshot with profile 1`] = `
<div>
  Profile page
</div>
`;
